apis/server/container_bridge.go

```diff
@@ -358,9 +358,6 @@ func (s *Server) statsContainer(ctx context.Context, rw http.ResponseWriter, req
 	name := mux.Vars(req)["name"]
 
 	var stream bool
-	if _, ok := req.Form["stream"]; !ok {
-		stream = true
-	}
 	stream = httputils.BoolValue(req, "stream")
```

Comment:
it seems that we can combine this line and the `var stream bool` into one line `:=`

Reply:
you are right.